styles/layouts/Error.module.scss

```diff
@@ -0,0 +1,22 @@
+.container {
+    @apply flex flex-col;
+    @apply size-full h-screen;
```

Comment:
Can we not use `h-screen` here, My problem with it is, this layout now doesnt work on a page with where its expected to fit into a grid, I think on the parent screen we should handle a container thats the height we need and have this size to meet that, alternatively in the parent you can apply the `100vh` class if needed. 

Seperation of things like this means we can change other components without breaking things.